.github/workflows/ci.yml

```diff
@@ -0,0 +1,37 @@
+name: Go
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set up Go 1.x
+      uses: actions/setup-go@v2
+      with:
+        go-version: ^1.15
+
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Dependencies
+      run: |
+        go get -v -t -d ./...
+        docker pull trufflesuite/ganache-cli
+
```

Comment:
We should also add `golangci-lint` here.

Reply:
With a configuration similar to our go-perun one.

Reply:
I used the default config now, should be good.